README.md

```diff
@@ -141,100 +126,82 @@ The following are advance examples for document string in `C++`.
 |:--------------------------------------|:-------------------------------------|
 | <img src="./etc/c++/cpp-struct.gif"/> | <img src="./etc/c++/cpp-class.gif"/> |
 
-## Document String
+## Documentation Strings
 
-You are able to customize default document string by tweaking these variables.
+You can customize default documentation strings by tweaking these variables.
 
 ### Type Name
 
-* `docstr-format-type` - default to `"{ %s }"`
+* `docstr-format-type` - default: `"{ %s }"`
 
-The default is wrap around curly brackets. It only takes one `%s` for type name
-to plug in.
+The default is wrap around curly brackets. It only takes one `%s` for the type name.
 
-* `docstr-show-type-name` - default to `t`
+* `docstr-show-type-name` - default: `t`
 
-If you don't want the type name to be shown; then set `docstr-show-type-name`
-to `nil` will make the trick. For instance, if you don't want the type name
-to be shown in `java-mode`; do the following.
+If you don't want the type name to be shown; then setting `docstr-show-type-name` to `nil` will do the trick. For instance, if you don't want the type name to be shown in `java-mode` do the following.
 
 ```el
 (add-hook 'java-mode-hook (lambda () (setq-local docstr-show-type-name nil)))
 ```
 
-* `docstr-default-typename` - default to `typename`
+* `docstr-default-typename` - default: `typename`
 
-You can change this value if you don't like the default type name. This
-variable is generally use for programming languages that aren't strong type.
-Like `Python`, `JavaScript`, `PHP`, etc.
+You can change this value if you don't like the default type name. This variable is generally used for programming languages that aren't statically typed like `Python`, `JavaScript`, `PHP`, etc.
```

Comment:
I think it static typing is meant here instead of strongly typed - Python is strongly typed :slightly_smiling_face:

Reply:
Ah, yes. I think you are correct. :)